qtsystems checksum failure

I am getting a failure on the `qtsystems` download checksum.

```bash
<snipped script output>
cd mxe
++ cd mxe
export PACKAGES="gcc openssl qt5"
++ export 'PACKAGES=gcc openssl qt5'
++ PACKAGES='gcc openssl qt5'
make MXE_TARGETS='i686-w64-mingw32.static' ${PACKAGES}
++ make MXE_TARGETS=i686-w64-mingw32.static gcc openssl qt5
[using autodetected 4 job(s)]
[build]     mxe-conf                 x86_64-unknown-linux-gnu                                             
[download]  lcms                                                                                          
[download]  qtsystems                                                                                     
[download]  qtvirtualkeyboard                                                                             

Download failed or wrong checksum of package qtsystems!
------------------------------------------------------------
  1000K .......... .......... .......... .......... .......... 78% 3.02M 0s
  1050K .......... .......... .......... .......... .......... 82%  145M 0s
  1100K .......... .......... .......... .......... .......... 86% 95.8M 0s
  1150K .......... .......... .......... .......... .......... 90% 97.2M 0s
  1200K .......... .......... .......... .......... .......... 93%  102M 0s
  1250K .......... .......... .......... .......... .......... 97% 2.77M 0s
  1300K .......... .......... .......... ..                   100% 1.56M=1.0s
2016-06-28 09:51:27 (1.25 MB/s) - written to stdout [1364068/1364068]
------------------------------------------------------------
[log]      /epc/t/262/hw-build/sysroot-linux-32/native/src/mxe/log/qtsystems-download

Makefile:451: recipe for target 'download-only-qtsystems' failed
make: *** [download-only-qtsystems] Error 1
make: *** Waiting for unfinished jobs....
[done]      mxe-conf                 x86_64-unknown-linux-gnu                              324 KiB        0m8.887s
altendky@tp:/epc/t/262/hw$
```

I was able to 'fix' it with this change though I don't know if the error is in the downloaded file or the reference checksum.

```diff
altendky@tp:/epc/t/262/hw-build/sysroot/native/src/mxe$ git rev-parse HEAD
4824bdea11b9627bb85e058550f1d46d2ccb5d1d
altendky@tp:/epc/t/262/hw-build/sysroot/native/src/mxe$ git diff
diff --git a/src/qtsystems.mk b/src/qtsystems.mk
index 03867e8..1fb62ee 100644
--- a/src/qtsystems.mk
+++ b/src/qtsystems.mk
@@ -3,7 +3,7 @@
 PKG             := qtsystems
 $(PKG)_IGNORE   :=
 $(PKG)_VERSION  := 4e3a7ed
-$(PKG)_CHECKSUM := 4aceb62b7e3135a1f29a31d71467d1c519ee25c36e1aa0b96b72540af5b198e3
+$(PKG)_CHECKSUM := 710d2b80f9125fb03fdb67dd6f3c4dd51396981e812fd4a3ea99b1b1290853e5
 $(PKG)_SUBDIR   := qtproject-$(PKG)-$($(PKG)_VERSION)
 $(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
 $(PKG)_URL      := https://github.com/qtproject/qtsystems/tarball/$($(PKG)_VERSION)/$($(PKG)_FILE)
altendky@tp:/epc/t/262/hw-build/sysroot/native/src/mxe$
```

--- a/src/qtsystems.mk
+++ b/src/qtsystems.mk
@@ -3,8 +3,8 @@
 PKG             := qtsystems
 $(PKG)_IGNORE   :=
 $(PKG)_VERSION  := 4e3a7ed
-$(PKG)_CHECKSUM := 4aceb62b7e3135a1f29a31d71467d1c519ee25c36e1aa0b96b72540af5b198e3
-$(PKG)_SUBDIR   := qtproject-$(PKG)-$($(PKG)_VERSION)
+$(PKG)_CHECKSUM := 710d2b80f9125fb03fdb67dd6f3c4dd51396981e812fd4a3ea99b1b1290853e5
+$(PKG)_SUBDIR   := qt-$(PKG)-$($(PKG)_VERSION)
 $(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
 $(PKG)_URL      := https://github.com/qtproject/qtsystems/tarball/$($(PKG)_VERSION)/$($(PKG)_FILE)
 $(PKG)_DEPS     := gcc qtbase qtdeclarative qtxmlpatterns

